GitDiffMargin/Properties/AssemblyInfo.cs

```diff
@@ -32,3 +33,17 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("3.2.0.0")]
 [assembly: AssemblyFileVersion("3.2.0.0")]
+
+#if true // Use this block if Visual Studio 2010 still needs to be supported
+[assembly: Guid("c279cada-3e46-4971-9355-50f43053d1b3")]
+[assembly: ProvideBindingPath]

```

Comment:
@sharwell Git Diff Margin is using not that much extra assemblies which aren't coming with Visual Studio, to my humble understanding. We have MVVMLight and LibGit2Sharp. So we were somehow lucky then